js/renovation/ui/grids/data_grid/data_grid.tsx

```diff
@@ -291,7 +291,7 @@ export class DataGrid extends JSXComponent(DataGridProps) implements DataGridFor
   }
 
   @Method()
-  navigateToRow(key: any): void {
+  async navigateToRow(key: any): DxPromise {
```

Comment:
Remove async because async does not work correctly now.

Reply:
Why is not DxPromise<void> as in d.ts?

Reply:
There is ts lint rule `@typescript-eslint/promise-function-async`: "Functions that return promises must be async", so I can't avoid using it